docs/book/src/reference/envtest.md

```diff
@@ -97,12 +97,12 @@ You can use environment variables and/or flags to specify the `kubectl`,`api-ser
 ### Environment Variables
 
 | Variable name | Type | When to use |
```

Comment:
Also, regards the links we need to use an alias and add them at the bottom of the page for better maintainability. Example:

[text][alias]

[alias]: link -> this one is at the bottom of the file.